src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -175,20 +187,43 @@ private List<ElementInfos> getVoltageLevelsInfos(Network network, @NonNull List<
                 .toList();
     }
 
+    private List<ElementInfos> getHvdcLinesInfosByType(Network network, @NonNull List<String> substationsId, InfoTypeParameters infoTypeParameters, List<Double> nominalVoltages, HvdcConverterStation.HvdcType type) {
+        Stream<HvdcLine> hvdcLines = substationsId.isEmpty()
```

Comment:
```suggestion
        Stream<HvdcLine> hvdcLineStream = substationsId.isEmpty()
```
I know this is existing code but better to rename with Stream as the above function get ids

Reply:
why not
fixed